test/typescript/test/runner.test.ts

```diff
@@ -67,7 +63,9 @@ describe('should fail', async () => {
     expect(msg).toMatchSnapshot()
 
     expect(stderr).toContain('FAIL  fail.test-d.ts') // included in tsconfig
-    expect(stderr).toContain('FAIL  only.test-d.ts') // .only
+
+    // TODO: Why should this be picked as well?
+    // expect(stderr).toContain('FAIL  only.test-d.ts') // .only
```

Comment:
Not sure why this started to fail now. The `only.test-d.ts` is not defined in the custom TS config.

Reply:
I don't remember 🤔